daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -773,6 +773,9 @@ private[lf] final class Compiler(
         compileBlock(bindings, body)
       case UpdateFetch(tmplId, coidE) =>
         FetchDefRef(tmplId)(compile(coidE))
+      case UpdateFetchInterface(_, _) =>
+        // TODO https://github.com/digital-asset/daml/issues/10810
+        sys.error("Interfaces not supported")
```

Comment:
fine to use FetchDefRef here

Reply:
I think I’d rather wait until that actually works and change it then. At the moment, that won’t do anything sensible.